authors/apps/favorite/views.py

```diff
@@ -0,0 +1,98 @@
+from rest_framework import status, exceptions
+from rest_framework import generics
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.views import Response
+from django.shortcuts import get_object_or_404
+
+from authors.apps.favorite.serializers import FavoriteSerializer
+from authors.apps.articles.models import Article
+from authors.apps.favorite.models import Favorite
+from authors.apps.favorite.messages import error_msgs, success_msg
+
+
+class FavoriteAPIView(generics.CreateAPIView):
+    """
+        A user should be able to add an article to their favorites
+    """
+    serializer_class = FavoriteSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, slug, *args, **kwargs):
+        """
```

Comment:
good work but we agreed to add a short description of what the method is doing. Do this for the rest of the methods